view/src/worker.rs

```diff
@@ -77,9 +81,12 @@ impl Worker {
         let specific_client =
             SpecificQueryServiceClient::connect(format!("http://{}:{}", node, pd_port)).await?;
 
-        let tm_client = tendermint_rpc::HttpClient::new(
-            format!("http://{}:{}", node, tendermint_port).as_str(),
-        )?;
+        let tm_client =
+            TendermintServiceClient::connect(format!("http://{}:{}", node, tendermint_port))
+                .await?;
+        let tm_proxy_client =
+            TendermintProxyServiceClient::connect(format!("http://{}:{}", node, tendermint_port))
+                .await?;
```

Comment:
these should be using `pd_port` now, right, since they can all be on the same endpoint?